### DISCUSSION TITLE
proteus libraries

### DISCUSSION THREAD

**Discussion by K4npai** [iTwin/itwinjs-core#2409] [Q&A] [answered]
Hi, I would like to know where I can find proteus libraries to be able to simulate microcontrollers including a raspberry pi, thanks

**Comment by pmconne** [comment, marked as answer]
Is [this](https://www.labcenter.com/) what you mean by "Proteus"? Please be more specific about what you need.

**Reply by K4npai** [reply]
yes, that´s what i mean, sorry for not being more specific

**Reply by pmconne** [reply]
To the best of my knowledge no such libraries exist.
